docs/platform/release-notes/tizen-5-5-m1.md

```diff
@@ -1,11 +1,26 @@
-# Tizen 5.5 Public M1 Release Notes
+# Tizen 5.5 Public M1
 
-Release Date: May. 31, 2019
+Release date: May 31, 2019
```

Comment:
Question

"Release date:"
 - Tizen 5.5 Public M2
 - Tizen 5.5 Public M1
 - Tizen 5.0 Public M2
 - Tizen 5.0 Public M1

"Release Date : "
 - Tizen 4.0 Public M3
 - Tizen 4.0 Public M2
 - Tizen 4.0 Public M1

Are we going to use small letter for "date" from 5.0  versions?